numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -39,7 +39,12 @@ def get(name):
 def add_newdoc(place, name, doc):
     doc = textwrap.dedent(doc).strip()
 
-    if name[0] != '_' and name != 'matmul':
+    skip = (
+        'matmul',
+        'clip',
+    )
+
+    if name[0] != '_' and name not in skip:
         # matmul is special, it does not use the OUT_SCALAR replacement strings
```

Comment:
Need to adjust the comment here on why `clip` is excluded; though perhaps better to put it behind each item in `skip`.